### DISCUSSION TITLE
Does MuliplayerSpawner and MultiplayerSynchonizer not work on server?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [godotengine/godot-proposals#4706] [Engine Core]
Does MuliplayerSpawner and MultiplayerSynchonizer not work on server?

In arcade / match making games the server can be a player.

**Comment by Calinou** [comment]
Please don't ask support questions here, as this issue tracker is meant to be used for feature proposals only. Use one of the other [community channels](https://godotengine.org/community) instead.
